@xen-orchestra/web/src/components/pool/PoolHeader.vue

```diff
@@ -30,7 +30,7 @@
     </RouterLink>
     <RouterLink v-slot="{ isActive, href }" :to="`/pool/${pool.id}/vms`" custom>
       <TabItem :active="isActive" :href tag="a">
-        {{ t('vms') }}
+        {{ t('vms', 0) }}
```

Comment:
According to Figma, show the vms variant.
```suggestion
      {{ t('vms', 2) }}
```